react/BottomSheet/BottomSheet.jsx

```diff
@@ -149,43 +159,47 @@ const BottomSheet = ({ toolbarProps, settings, children }) => {
   })
 
   return (
-    <MuiBottomSheet
-      peekHeights={peekHeights}
-      defaultHeight={initPos}
-      backdrop={false}
-      fullHeight={false}
-      onIndexChange={snapIndex => handleOnIndexChange(snapIndex)}
-      styles={{ root: styles.root }}
-      threshold={0}
-      // springConfig doc : https://docs.pmnd.rs/react-spring/common/configs
-      springConfig={{
-        tension: defaultBottomSheetSpringConfig.tension,
-        friction: defaultBottomSheetSpringConfig.friction,
-        clamp: defaultBottomSheetSpringConfig.clamp
-      }}
-      disabledClosing={true}
-      snapPointSeekerMode="next"
-    >
-      <div ref={innerContentRef}>
-        <Paper
-          data-testid="bottomSheet-header"
-          className="u-w-100 u-h-2-half u-pos-relative u-flex u-flex-items-center u-flex-justify-center"
-          ref={headerRef}
-          elevation={0}
-          square
-        >
-          <div style={styles.indicator} />
-        </Paper>
-        <Stack
-          style={styles.stack}
-          className="u-flex u-flex-column u-ov-hidden"
-          spacing="s"
-        >
-          {overriddenChildren}
-        </Stack>
-      </div>
-      <div style={{ height: bottomSpacerHeight }} />
-    </MuiBottomSheet>
+    <>
+      {isImmersive && <div style={styles.flagshipImmersive} />}
```

Comment:
let's use a span for that usage.

Reply:
y'a t-il également une modif dans le code plus bas ? Comme on a tout wrappé dans un `<></>` ce n'est pas évident à repérer...

Reply:
non rien d'autre